collections/_article/introducing-the-improved-ufbx-importer-in-godot-4-3.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Introducing the improved ufbx importer in Godot 4.3"
+excerpt: "Godot 4.3 now includes native .fbx support"
+categories: ["progress-report"]
+author: Lyuma
+image: /storage/blog/covers/PlaceHolder.webp
+date: 2099-09-09 00:00:00
+---
+
+Thanks to [a game-changing contribution by Samuli Raivio (bqqbarbhg) and Ernest Lee (fire)](https://github.com/godotengine/godot/pull/81746), Godot 4.3 now includes native .fbx support. This was made possible using Samuli’s free and open source [ufbx single-file fbx library(https://github.com/ufbx/ufbx). The ufbx importer will be used by default for all newly imported fbx files in Godot 4.3
```

Comment:
I thought it would be humorous, but I suppose this pun only works so many times in the Game industry before it gets old ;-) I'll change it to the last one.